chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -77,6 +77,23 @@ object CBPurchase {
         })
     }
 
+    fun changeProduct(
+        changeProductParams: ChangeProductParams, customer: CBCustomer? = null,
+        callback: CBCallback.PurchaseCallback<String>
+    ) {
+        this.customer = customer
+        changeProduct(changeProductParams, callback)
+    }
+
+    private fun changeProduct(changeProductParams: ChangeProductParams, callback: CBCallback.PurchaseCallback<String>) {
+        isSDKKeyValid({
+            log(customer, changeProductParams.purchaseProductParams.product.id)
```

Comment:
The `action` being logged in this method is `before_purchase_command`.
Can we change this to something appropriate?

Reply:
modified log method to accept name, oldproductId?, action. And pass those values based on where it is being called. Does this make sense?